crawler/utils/dockerutils.py

```diff
@@ -360,3 +363,31 @@ def get_docker_container_rootfs_path(long_id, inspect=None):
         raise DockerutilsException('Not supported docker storage driver.')
 
     return rootfs_path
+
+
+def poll_container_create_events(timeout=0.1):
+    try:
+        client = docker.Client(base_url='unix://var/run/docker.sock',
```

Comment:
If we want to use the same client, then we need to buffer these events ourselves..client won't store them.  Then, it again goes back to the design where we start a singleton monitor process with worker and share a queue with it..